src/rust-2024/cargo-table-key-names.md

```diff
@@ -0,0 +1,42 @@
+# Cargo: Table and key name consistency
+
+🚧 The 2024 Edition has not yet been released and hence this section is still "under construction".
+
+## Summary
+
+- Several table and key names in `Cargo.toml` have been removed where there was previously two ways to specify it.
+    - Removed `[project]`, use `[package]` instead.
+    - Removed `default_features`, use `default-features` instead.
+    - Removed `crate_type`, use `crate-type` instead.
+    - Removed `dev_dependencies`, use `dev-dependencies` instead.
+    - Removed `build_dependencies`, use `build-dependencies` instead.
```

Comment:
```suggestion
    - Removed `[project]`, use `[package]` instead.
    - Removed `default_features`, use `default-features` instead.
    - Removed `crate_type`, use `crate-type` instead.
    - Removed `proc_macro`, use `proc-macro` instead.
    - Removed `dev_dependencies`, use `dev-dependencies` instead.
    - Removed `build_dependencies`, use `build-dependencies` instead.
```

Reply:
(proc_macro)

Reply:
Thanks! Not sure how I missed it since it was in my notes.